tests/unit/amo/components/TestCollection.js

```diff
@@ -663,10 +657,35 @@ describe(__filename, () => {
       'pathname',
       `/collections/${username}/${slug}/`,
     );
-    expect(paginator).toHaveProp('queryParams', filters);
+    expect(paginator).toHaveProp('queryParams', queryParams);
     expect(wrapper.find('.Collection-edit-link')).toHaveLength(0);
   });
 
+  it('renders a sort card', () => {
+    const sort = COLLECTION_SORT_NAME;
+
+    const wrapper = renderComponent({
+      location: fakeRouterLocation({ query: { collection_sort: sort } }),
+    });
+
+    const sortOptions = wrapper.instance().sortOptions();
+
+    expect(wrapper.find('.Collection-sort')).toHaveLength(1);
+    expect(wrapper.find('.Sort-label')).toHaveText('Sort add-ons by');
+    expect(wrapper.find('.Sort-select')).toHaveProp('defaultValue', sort);
+    expect(wrapper.find('.Sort-select')).toHaveProp(
+      'onChange',
+      wrapper.instance().onSortSelect,
+    );
+    wrapper
+      .find('.Sort-select')
+      .children()
+      .forEach((option, index) => {
```

Comment:
This is a risky pattern because if the array has zero items, the test will pass. To make it fail, you'd write it more like:

```js
const options = wrapper.find('.Sort-select').children();

sortOptions.forEach((item, index) => {
  expect(options.at(index)).toHaveProp('value', item.value);
  expect(options.at(index)).toHaveText(item.children);
});
```

That is, the `sortOptions` array is under your control so it should lead the assertion.